downstream/modules/platform/proc-installing-the-aap-setup-bundle.adoc

```diff
@@ -5,76 +5,25 @@
 
 
 .Procedure
 
-. Download the {PlatformNameShort} setup bundle package by navigating to the link:{PlatformDownloadUrl}[{PlatformName} download] page and clicking btn:[Download Now] for the {PlatformNameShort} {PlatformVers} Setup Bundle.
+. Download the {PlatformNameShort} setup bundle package by navigating to the link:https://access.redhat.com/downloads/content/480/ver=2.4/rhel---9/2.4/x86_64/product-software{PlatformDownloadUrl}[{PlatformName} download] page and clicking btn:[Download Now] for the {PlatformNameShort} {PlatformVers} Setup Bundle.
 
-. From {ControllerName}, untar the bundle:
+. On control node, untar the bundle:
 +
 ----
 $ tar xvf \
-   ansible-automation-platform-setup-bundle-2.4-1.tar.gz
-$ cd ansible-automation-platform-setup-bundle-2.4-1
-----
-+
-. Edit the inventory file to include the required options:
-
-.. automationcontroller group
-.. automationhub group
-.. admin_password
-.. pg_password
-.. automationhub_admin_password
-.. automationhub_pg_host, automationhub_pg_port
-.. automationhub_pg_password
-+
-*Example Inventory file*
-+
-----
-[automationcontroller]
-automationcontroller.example.org ansible_connection=local
-
-[automationcontroller:vars]
-peers=execution_nodes
-
-[automationhub]
-automationhub.example.org
-
-[all:vars]
-admin_password='password123'
-
-pg_database='awx'
-pg_username='awx'
-pg_password='dbpassword123'
-
-receptor_listener_port=27199
-
-automationhub_admin_password='hubpassword123'
-
-automationhub_pg_host='automationcontroller.example.org'
-automationhub_pg_port=5432
-
-automationhub_pg_database='automationhub'
-automationhub_pg_username='automationhub'
-automationhub_pg_password='dbpassword123'
-automationhub_pg_sslmode='prefer'
+   ansible-automation-platform-setup-bundle-2.5-1.tar.gz
+$ cd ansible-automation-platform-setup-bundle-2.5-1
 ----
 +
-. Run the {PlatformNameShort} setup bundle executable as the root user:
-+
-----
-$ sudo -i
-# cd /path/to/ansible-automation-platform-setup-bundle-2.4-1
-# ./setup.sh
-----
-+
-. When installation is complete, navigate to the Fully Qualified Domain Name (FQDN) for the {ControllerName} node that was specified in the installation inventory file.
-
-. Log in using the administrator credentials specified in the installation inventory file.
+. Edit the inventory file to include variables based on your host names and desired password values.
 
 [NOTE]
 ====
-The inventory file must be kept intact after installation because it is used for backup, restore, and upgrade functions. Keep a backup copy in a secure location, given that the inventory file contains passwords.
+See section link:https://docs.google.com/document/d/1fIrlSrj8bSSWBVptXWmm0ZFixhZ4mM5pW2CQx_3Oh7A/edit#heading=h.tevmra11zce4[3.2 Inventory file examples base on installation scenarios] for a list of examples that best fits your scenario.
```

Comment:
I don't think we can use the Google docs link - we need the link for the customer portal